europe/README.EUROPE

```diff
@@ -294,3 +294,111 @@ Residuals at 130 further test points average 0.06m, maximum 0.16m.
+
+*Source*: [National Land Survey of Iceland](https://atlas.lmi.is/LmiData/index.php?id=626468364600)
+*Format*: NTv2
+*License*: [Creative Commons Attribution 4.0](https://creativecommons.org/licenses/by/4.0/)
+*Credit*: (c)2017-2019 Grid created by Guðmundur Valsson.
+
+The transformation is designed so that the ISN2016 points should fit within 1-2 cm accuracy, but in the area around the earthquake area in the South and North of Vatnajökull the accuracy is worse.
+
+Recommended to transform from ISN2004 2D geodetic (EPSG:5324) to ISN2016 2D geodetic (EPSG:8086), also this transformation has code in EPSG Dataset (EPSG:9233)
+
+* ISN2004_ISN2016.gsb
+
+
+
+### Iceland: Deformation model: ISN93 -> ISN2016 
+
+*Source*: [IGN France](https://geodesie.ign.fr/index.php?page=grilles)   
+*Format*: NTv2,GTX  
+*License*: [Creative Commons Attribution 4.0](https://creativecommons.org/licenses/by/4.0/)
+*Credit*: (c)2017-2019 Grid created by Guðmundur Valsson.
+
+Deformation model used in transformation between ISN93 reference frame and ISN2016 reference frame
+Horizontal transformation is designed so that the ISN2016 points should fit within 1-2 cm accuracy, but in the area around the earthquake area in the South and North of Vatnajökull the accuracy is worse.
+Vertical transformation 
+Recommended to transform from ISN93 3D geodetic (EPSG:4945) to ISN2016 3D geodetic (EPSG:8085)
+
+See the `ISL` init-file for examples of use.
+
+* ISN93_ISN2016.gsb
+* ISN93_ISN2016.gtx
+
+
+### Iceland: Deformation model: ISN2004 -> ISN2016 
+
+*Source*: [IGN France](https://geodesie.ign.fr/index.php?page=grilles)   
+*Format*: NTv2,GTX  
+*License*: [Creative Commons Attribution 4.0](https://creativecommons.org/licenses/by/4.0/)
+*Credit*: (c)2017-2019 Grid created by Guðmundur Valsson.
+
+Deformation model used in transformation between ISN2004 reference frame and ISN2016 reference frame
+Horizontal transformation is designed so that the ISN2016 points should fit within 1-2 cm accuracy, but in the area around the earthquake area in the South and North of Vatnajökull the accuracy is worse.
+Vertical transformation 
+Recommended to transform from ISN2004 3D geodetic (EPSG:5323) to ISN2016 3D geodetic (EPSG:8085)
+
+See the `ISL` init-file for examples of use.
+
+* ISN2004_ISN2016.gsb
+* ISN2004_ISN2016.gtx
+
+
+### Iceland: ISN2004 ellipsoidal height  -> ISH2004 vertical datum
+
+*Source*: [National Land Survey of Iceland](https://atlas.lmi.is/LmiData/index.php?id=626468364600)
+*Format*: GTX  
+*License*: [Creative Commons Attribution 4.0](https://creativecommons.org/licenses/by/4.0/)
+*Credit*: (c)2017-2019 Grid created by Guðmundur Valsson.
+
+Geoid usually fits within 1-2cm, but in some areas it‘s slightly worse 3-5cm. The std for the fitted geoid is 1.7cm based on over 300 gnss levellin points
+
+Grid to convert from ISN2004 ellipsoidal height (EPSG:5323) to national vertical datum ISH2004 (EPSG:8089)
```

Comment:
not yet registered in EPSG ? You could potentially add a custom entry in the meantime as done in https://github.com/OSGeo/PROJ/pull/1759/files#diff-7d4dfd28cbd3f979bcc5f926e930938fR97

Reply:
actually thinking that a better place for custom transformations is in the grid_transformation_custom.sql file at https://github.com/OSGeo/PROJ/blob/master/data/sql/grid_transformation_custom.sql . You will note the use of a 'GravityRelatedHeight to Geographic3D' PROJ-specific method that inverts the confusing convention used by the EPSG's 'Geographic3D to GravityRelatedHeight'. As the .gtx file contains the undulation and all other grid transformations use an additive convention (to go from source to target, one adds the value(s) of the grid), this is far more natural.

Reply:
I added entries into grid_transformation_custom.sql